src/store/tests/ducks/messages.ts

```diff
@@ -0,0 +1,70 @@
+import { ActionCreator, AnyAction, Store } from 'redux';
```

Comment:
@moshthepitt @KipSigei do you think it is possible to use [reducer factory](https://www.npmjs.com/package/@opensrp/reducer-factory) here?

Reply:
I think this would be at Kip's discretion since this is a test

Reply:
created a separate issue for that [here](https://github.com/OpenSRP/web/issues/27)